package/boot/uboot-mvebu/patches/220-fix-build-without-libcrypto.patch

```diff
@@ -0,0 +1,31 @@
+--- a/tools/Makefile
++++ b/tools/Makefile
+@@ -96,7 +96,8 @@ AES_OBJS-$(CONFIG_TOOLS_LIBCRYPTO) := $(
+ 
+ # Cryptographic helpers that depend on openssl/libcrypto
+ LIBCRYPTO_OBJS-$(CONFIG_TOOLS_LIBCRYPTO) := $(addprefix lib/, \
+-					fdt-libcrypto.o)
++					fdt-libcrypto.o) \
++					kwbimage.o
+ 
+ ROCKCHIP_OBS = lib/rc4.o rkcommon.o rkimage.o rksd.o rkspi.o
+ 
+@@ -117,7 +118,6 @@ dumpimage-mkimage-objs := aisimage.o \
+ 			imximage.o \
+ 			imx8image.o \
+ 			imx8mimage.o \
+-			kwbimage.o \
+ 			lib/md5.o \
+ 			lpc32xximage.o \
+ 			mxsimage.o \
+@@ -169,8 +169,8 @@ HOST_EXTRACFLAGS	+= -DCONFIG_FIT_SIGNATU
+ HOST_EXTRACFLAGS	+= -DCONFIG_FIT_CIPHER
+ endif
+ 
+-# MXSImage needs LibSSL
+-ifneq ($(CONFIG_MX23)$(CONFIG_MX28)$(CONFIG_ARMADA_38X)$(CONFIG_TOOLS_LIBCRYPTO),)
++# MXSImage needs LibSSL <- Nope! Read the frogging notice at the top
++ifneq ($(CONFIG_TOOLS_LIBCRYPTO),)
+ HOSTCFLAGS_kwbimage.o += \
+ 	$(shell pkg-config --cflags libssl libcrypto 2> /dev/null || echo "")
+ HOSTLDLIBS_mkimage += \
```

Comment:
What such of shit is this?? Ah, again idiot @dhewg... another attempt to break something which is working.

I really cannot understand how openwrt can again accept some total nonsense patch, which was not reviewed, which is fully wrong and again without any description...

Why is description missing? Oh, because writing into description that author's intention is only to break mvebu is not so cool... And because @dhewg is known to include into openwrt any patch which is marked as crap or not suitable into upstream.

openwrt devs: is your direction only kill mvebu platform by including random craps???

Reply:
obsolete patch removed...
thanks !